waveguide_mode_converter/README.md

```diff
@@ -0,0 +1,22 @@
+# Waveguide Mode Converter
+
+This is a waveguide mode converter test problem. The design objective is to maximize the conversion of power from the
+fundamental waveguide mode of the input waveguide to the second-order mode of the output waveguide. The geometry is two-dimensional.
```

Comment:
Be explicit about the objective function.

Reply:
Done.